src/ctapipe/image/tests/test_extractor.py

```diff
@@ -281,6 +282,17 @@ def test_extract_around_peak_charge_expected():
     assert_equal(charge, n_samples)
 
 
+def test_variance_extractor(toymodel):
+    _, subarray, _, _, _, _ = toymodel
+    # make dummy data with known variance
+    rng = np.random.default_rng(0)
+    var_data = rng.normal(2.0, 5.0, size=(5000, 2, 1855))
+    extractor = ImageExtractor.from_name("VarianceExtractor", subarray=subarray)
+
+    variance = extractor(var_data, 0, None, None).image
+    assert np.isclose(variance, np.var(var_data, axis=2)).all()
```

Comment:
Use `np.testing.assert_all_close(variance, np.var(...))` for better error messages in case of failure